src/cider/exec/nextgen/context/RuntimeContext.h

```diff
@@ -78,6 +80,7 @@ class RuntimeContext {
   std::vector<std::pair<CodegenContext::BatchDescriptorPtr, BatchPtr>> batch_holder_;
   std::vector<std::pair<CodegenContext::BufferDescriptorPtr, BufferPtr>> buffer_holder_;
   std::shared_ptr<StringHeap> string_heap_ptr_;
+  CodegenContext::HashTableDescriptorPtr hashtable_holder_;
```

Comment:
Shouldn't the `hashtable_holder_` hold a pair with descriptor and hash_table ptr, similar like `batch_holder_`? @bigPYJ1151